packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -581,23 +587,26 @@ class _DropdownButtonState<T> extends State<DropdownButton<T>> with WidgetsBindi
       style: _textStyle,
       child: new SizedBox(
         height: widget.isDense ? _denseButtonHeight : null,
-        child: new Row(
-          mainAxisAlignment: MainAxisAlignment.spaceBetween,
-          mainAxisSize: MainAxisSize.min,
-          children: <Widget>[
-            // If value is null (then _selectedIndex is null) then we display
-            // the hint or nothing at all.
-            new IndexedStack(
-              index: _selectedIndex ?? hintIndex,
-              alignment: FractionalOffset.centerLeft,
-              children: items,
-            ),
-            new Icon(Icons.arrow_drop_down,
-              size: widget.iconSize,
-              // These colors are not defined in the Material Design spec.
-              color: Theme.of(context).brightness == Brightness.light ? Colors.grey.shade700 : Colors.white70
-            ),
-          ],
+        child: new ExcludeSemantics(
+          excluding: _dropdownRoute != null,
```

Comment:
Doesn't look like that's the case, because the test fails if I hard-code it to `false`. Perhaps I should add ExcludeSemantics somewhere at the route level?